test/header.js

```diff
@@ -1,166 +0,0 @@
-/* Copyright 2020 Record Replay Inc. */
```

Comment:
I forgot to add in my review, but this file is actually used by the backend for static live tests.  It would be difficult to rework those tests to use playwright currently for a couple reasons:

1. We don't release a playwright windows build currently.
2. Playwright builds are generated from a branch that is merged from gecko's webreplay-release branch, so generating playwright builds for gecko changes that haven't merged to that branch isn't possible yet.  Our CI needs to run the static live tests on gecko branches.

I don't see an easy fix for 2.  I think the best fix would be to add this to the backend (cutting the dependency) before removing it from devtools.